lightning/restore/tidb.go

```diff
@@ -168,10 +171,17 @@ func InitSchema(ctx context.Context, g glue.Glue, database string, tablesSchema
 				ctx,
 				s,
 				"create table",
-				logger.With(zap.String("table", common.UniqueTable(database, tbl))),
+				logger.With(zap.String("table", common.UniqueTable(database, tblSchema.tblName))),
 			)
+			failpoint.Inject("sqlCreateStmts", func() {
+				if i == 0 {
```

Comment:
if the count matters, I think you can add an int parameter to this fail-point to determine at which condition, it should return an error